modules/dreamview/backend/teleop/teleop.h

```diff
@@ -51,11 +53,36 @@ class TeleopService {
   WebSocketHandler *websocket_;
 
 #ifdef TELEOP
+
+  // modem info readers and callback
+  std::shared_ptr<cyber::Reader<modules::car1::network::ModemInfo>>
+      modem0_info_reader_;
```

Comment:
i thought dreamview backend will listen to a combined modem signal summary instead of individual ones? Or does the modem monitor node computes the ranking together but still set the ranking info in separate modem signals?

Reply:
At this moment each modem has a channel (ModemInfo), and its rank is in the message.
It sort of works, but it has limitations (for example, a fleet of cars where some cars don't have 3 modems).
We can combine this into a single channel with a repeated field, which is closer to what the original backend expected. This is probably something I would consider doing even if you hadn't brought it up.